substrate/frame/staking-async/src/session_rotation.rs

```diff
@@ -378,8 +355,8 @@ impl<T: Config> Eras<T> {
 #[cfg(any(feature = "try-runtime", test, feature = "runtime-benchmarks"))]
 #[allow(unused)]
 impl<T: Config> Eras<T> {
-	/// Ensure the given era is present, i.e. has not been pruned yet.
-	pub(crate) fn era_present(era: EraIndex) -> Result<(), sp_runtime::TryRuntimeError> {
+	/// Ensure the given era's data is fully present (all storage intact and not being pruned).
+	pub(crate) fn era_fully_present(era: EraIndex) -> Result<(), sp_runtime::TryRuntimeError> {
```

Comment:
Are we checking all era based storages here?

Reply:
Its checking `ErasValidatorPref`, `ErasStakersOverview`, `ErasTotalStake`, `ErasValidatorReward`. Not checking `ClaimedRewards`, `ErasStakersPaged`, `ErasRewardPoints`. Or am I misunderstanding something?